### PR TITLE
Fixed Product form when stock management is disabled

### PR DESCRIPTION
**Pull request by mickaelandrieu**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | 1.7.4.x
| Description?  | The product form was broken when stock management feature was disabled
| Type?         | bug fix
| Category?     | BO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | http://forge.prestashop.com/browse/BOOM-6000
| How to test?  | See ticket => http://forge.prestashop.com/browse/BOOM-6000

#### Important guidelines

* Make sure [your local branch is up to date](https://help.github.com/articles/syncing-a-fork/) before commiting your changes!
* Your code MUST respect [our Coding Standards](http://doc.prestashop.com/display/PS16/Coding+Standards) (for code written in PHP, JavaScript, HTML/CSS/Smarty/Twig, SQL)!

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/9327)

### CODE REVIEW

**Review by PierreRambaud — APPROVED** [review]

**Comment by mickaelandrieu** [comment]
@PierreRambaud are we waiting for one more Approval to add ``Waiting for QA`` label?

**Comment by PierreRambaud** [comment]
@mickaelandrieu Missing add the Waiting for QA. :P And I said the double review is needed when you have many files edited =)

**Comment by eternoendless** [comment]
Thank you @mickaelandrieu

**Pull request merged**
